### DISCUSSION TITLE
v1.4.19

### DISCUSSION THREAD

**Discussion by sgkoishi** [sgkoishi/PackTracker#41] [General]
* Update 27.6.2.188645.187190
  * Add `922 Showdown in the Badlands` pack
  * Add `944 Wild West 2` pack
  * Add `952 Golden Showdown in the Badlands` pack

It should be available a few days earlier 🤔 

<hr /><em>This discussion was created from the release <a href='https://github.com/sgkoishi/PackTracker/releases/tag/v1.4.19'>v1.4.19</a>.</em>

**Comment by ZlatenDC** [comment]
Hi! Thanks for your work developing and maintaining this plugin. Will it be possible to track catch-up packs?

**Reply by sgkoishi** [reply]
~~Does it work?~~

**Comment by sgkoishi** [comment]
I have no idea. It seems that the "Wild West 2" is the first catch-up pack

**Reply by ZlatenDC** [reply]
Thanks so much! Fingers crossed it is possible.

**Reply by sgkoishi** [reply]
Sadly, no. From the animation, it's kinda similar to the multi-pack opening 🤔

**Reply by ZlatenDC** [reply]
Thanks for looking into it. I thought it would be difficult when seeing it yesterday too.
